test-vr/README.md

```
@@ -152,6 +152,32 @@ Docker is the recommended environment for running visual regression tests and up
+## Running tests in UI mode
+
+`npm run test-vr:ui` opens the Playwright UI at http://localhost:8080 and also exposes the
+story gallery at http://localhost:3100/gallery/index.html.
+
+The gallery dev server is started once per container session (`test-vr/.bin/playwright-test`
+pre-starts it), so reloading the UI or opening a second tab is safe. Prefer running a single
+spec file or test, or the `chromium` project: running all projects is long and CPU heavy.
+
+## Error: Cannot find module '@playwright/test' (or 'vite')
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)